app/modules/sightings/models.py

```diff
@@ -799,28 +864,37 @@ def send_identification(
                         )
                     else:
                         log.warning(
-                            f'Sage Identification on Sighting({self.guid}) Job({job_uuid}) failed to start '
+                            f'{debug_context} Sage Identification failed to start '
                             f'code: {ex.status_code}, acm_status_code: {acm_status_code}, giving up'
                         )
-                        self.stage = SightingStage.failed
-
+                        self.jobs[job_uuid_str]['active'] = False
+                        self.jobs[job_uuid_str]['success'] = False
+                        self.jobs[job_uuid_str]['failure_reason'] = 'too many retries'
         else:
-            # TODO, this is correct for MVP as there is only one id per Sighting but this will need
-            # rework when there are multiple
-            self.stage = SightingStage.un_reviewed
+            self.jobs[job_uuid_str]['active'] = False
+            self.jobs[job_uuid_str]['success'] = False
+            self.jobs[job_uuid_str]['failure_reason'] = 'No ID request built'
+
+        # This is necessary because we can only mark self as modified if
+        # we assign to one of the database attributes
+        self.jobs = self.jobs
+        with db.session.begin(subtransactions=True):
+            db.session.merge(self)
 
     # validate that the id response is a valid format and extract the data required from it
     def _parse_id_response(self, job_id_str, data):
         from app.extensions.acm import from_acm_uuid
 
         status = data.get('status', 'failed')
-
+        result = {
+            'scores_by_annotation': [],
+            'scores_by_individual': [],
+        }
         if status != 'completed':
-            self.stage = SightingStage.failed
```

Comment:
Yes this was a bug which assumed there was only one ID job. The stage setting has been moved out to the calling code which handles all jobs as opposed to this function which should not be changing stuff.